azerothcore/azerothcore-wotlk
# fix(DB/Object): Reduce dark iron pillow respawn time

## Changes Proposed:
-  reduces the respawn time of dark iron pillows
-  

## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/7022

## SOURCE:


## Tests Performed:

- None


## How to Test the Changes:


1. Teleport to a dark iron pillow
2. Open it with the quest (look at the issue)
3. Check the respawn time

## Known Issues and TODO List:

No


## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
